src/plugins/data-card/data-card-tile.tsx

```diff
@@ -139,20 +140,14 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
   const duplicateCard = () => {
     const originalCaseIndex = content.caseIndex;
     const copyableCase = content.caseByIndex(originalCaseIndex);
-    const desiredIndex = originalCaseIndex + 1;
-    const beforeId = content.dataSet.caseIDFromIndex(desiredIndex);
-
-    content.addNewCaseFromAttrKeys(content.existingAttributes(), beforeId);
-    const newCaseId = content.dataSet.caseIDFromIndex(desiredIndex);
-
-    if (newCaseId && content.isEmptyCase(newCaseId)){
-      for (const attrId in copyableCase){
-        const foundValue = copyableCase[attrId] || "";
-        const copyableValue: string = foundValue as string;
-        content.setAttValue(newCaseId, attrId, copyableValue);
-      }
+    if (copyableCase) {
+      // strip __id__ so a new id will be generated on insertion
```

Comment:
Oh, I see.  Thanks, I could not figure out how to do this without creating an empty and collecting its id.